src/pages/user/post-settings.tsx

```diff
@@ -1,7 +1,9 @@
+import { PostSettingsPage } from "../../features/user/components/post-settings-page"
+
 export const PostSettings = () => {
```

Comment:
To ten plik powinien się nazywać PostSettingsPage, a komponent który on zwraca PostSettings

Reply:
Myślę że jak PostSettings znajduje się w folderze Page to wiadomo że jest Page. Ale faktycznie te plik, który importuje mógłbym lepiej nazwać.